build.gradle

```diff
@@ -155,7 +156,8 @@ task systemtest(dependsOn: 'zipReport', type: Test) {
     }
 }
 
-task startServerInBackground(dependsOn: 'classes', type: com.github.psxpaul.task.JavaExecFork) {
+task startServerInBackground(dependsOn: ['zipReport', 'compileJava', 'classes'], type: com.github.psxpaul.task.JavaExecFork) {
```

Comment:
The name of this gradle task was actually a little misleading to me before. It seems like a general server start command for any report (that was already generated). It seems however that it's only really used for the Cypress tests.

Would something like `serveTestReportInBackground` be a more accurate description?